### PR TITLE
PCC02 niilz_stroh

### PR DESCRIPTION
**Pull request by niilz**
Difficulty level (1-10): [5]
Estimated time spent (hours): [2]
Completed (yes/no): [Yes]
I stretched my coding skills (if yes what did you learn?): [list comprehensions with two loops and practicing the usage of "all()"]
Other feedback (what can we improve?): [i still do not really understand how to check if my solutions is ok. Or: what I should compare it to]

### CODE REVIEW

File: 02/niilz_stroh/game.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+def get_7_chars():
+    return [POUCH[rd.randint(0, len(POUCH))].lower() for _ in range(7)]
+
+def possible_words(letters):
+    words = []
+    for word in DICTIONARY:
+        if not all([letter in letters for letter in word]): continue
+        else: words.append(word)
+    words = [word for word in words if all([word.count(char) <= letters.count(char) for char in word])]
```
Comment (pybites) [inline 02/niilz_stroh/game.py]: Consider splitting over multiple lines to not hit PEP E501

File: 02/niilz_stroh/game.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+
+def main():
+    letters = get_7_chars()
+    possible_w = possible_words(letters)
+    user_word = input(f"form a word from the letters '{letters}'")
+    if not all([user_word.count(char) <= letters.count(char) for char in user_word]):
+        print("sorry but have used some chars more often than they were given")
+    else:
+        print(f"your word scored {calc_word_value(user_word)} points.\n\
```
Comment (pybites) [inline 02/niilz_stroh/game.py]: Good reminder about \
I usually split long strings over multiple lines using:
~~~
print('line1'
         'line2')
~~~

Comment (niilz) [reply 02/niilz_stroh/game.py]: OH MY you are actually giving REAL advice!! Code-Review, encouraging tips and everything.
THAT IS FANTASTIC!!!! Thank you so much. I honestly did not expect that, when I signed up. 
Highly appreciated.

Comment (bbelderbos) [reply 02/niilz_stroh/game.py]: Yeah just started doing that ... I don't have time to run the solution, but at least I can quickly look over the code and comment :)
Glad it was useful, keep coding and learning!

**Pull request merged**